datumaro/components/visualizer.py

```diff
@@ -337,7 +391,18 @@ def _draw_super_resolution_annotation(
         ax: Axes,
         context: List,
     ) -> None:
-        raise NotImplementedError(f"{ann.type} is not implemented yet.")
+        assert (
+            len(context) == 0
+        ), "It cannot visualize more than one SuperResolutionAnnotation per item."
+
+        warnings.warn(
+            "SuperResolutionAnnotation overdraw high resolution image on top of the original image. "
```

Comment:
Please correct this to "SuperResolutionAnnotation overdraws the high-resolution image over the original image."

Reply:
Fixed.